app/views.py

```diff
@@ -310,7 +331,8 @@ class SmsView(ListCreateAPIView):
     def get_queryset(self):
         user = self.request.user
         if user.role == USER_TYPE_DHO:
-            model = SmsModel.objects.filter(sender_id=user.id).order_by('-created_at')
+            model = SmsModel.objects.filter(
```

Comment:
You should not be editing this file

Reply:
I think when I formatted the document. some work disappeared.  i never met to do that